packages/core/src/modules/credentials/repository/CredentialRecord.ts

```diff
@@ -43,6 +47,8 @@ export type DefaultCredentialTags = {
   connectionId?: string
   state: CredentialState
   credentialId?: string
+  revocationRegistryId?: string
+  credentialRevocationId?: string
```

Comment:
Oh maybe we can call this `indyRevocationRegistryId` and `indyCredentialRevocationId` to not make it harder to use a generic name for non-indy revocation formats in the future? Or do you think this way we'll be able to reuse the name for other formats? In that case also fine, but not sure how applicable these parameters are to other revocation methods.

Reply:
@TheTreek are these needed in the tags here since we've got this information in the metadata now?

Reply:
>@TheTreek are these needed in the tags here since we've got this information in the metadata now?

These are needed in the tags so we can search for the credential by them. We don't have that ability if it is solely in metadata.

Reply:
Got it--I'd agree with Timo's suggestion to use `indyRevocationRegistryId` and `indyCredentialRevocationId` since this is somewhat specific to Indy. 

> Or do you think this way we'll be able to reuse the name for other formats

There might be overlap, however I don't think we have enough info on other methods right now to make an educated decision, so I think doing Indy specific names will give us the most flexibility in the future IMO.